.gitignore

```diff
@@ -1,5 +1,5 @@
 rsconnect/
-*.sqlite
+.sqlite
```

Comment:
Please adjust this to `/*.sqlite` as seen [here](https://github.com/pharmaR/risk_assessment/pull/312/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L2-R2). This tells git to restrict the ignore to the main directory and not subdirectories. That way the credentials, credentials_dev, and database SQLite files are not being added to the tracking. For reference: https://git-scm.com/docs/gitignore.

Reply:
OK on the changes above

Reply:
For some reason those files are still being tracked...

Reply:
Appears you actually need to remove it from the cache: https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore#:~:text=on%20this%20post.-,.,that%20are%20already%20being%20tracked.&text=The%20removal%20of%20the%20file,happen%20on%20the%20next%20commit.

Evidently `.gitignore` only applies to files that are not being tracked.

Reply:
OK done.
I ran `git rm -r --cached . && git add . && git commit -am "Remove ignored files"`
and pushed the change